.pre-commit-config.yaml

```diff
@@ -44,6 +44,15 @@ repos:
         pass_filenames: false
         require_serial: true
 
+-   repo: local
+    hooks:
+      - id: build-installer
```

Comment:
Nit: since it's a local hook, you can put it inside the previous one. We should move all the local hooks under a single "repo: local". Followup PR